src/adv.py

```diff
@@ -49,3 +53,61 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+
```

Comment:
I like how you set this up in seperate functions like this.  Its very easy to read.  I haven't ran it but looks like it works and all the code is very readable.  Great work!